rc/filetype/git.kak

```diff
@@ -43,6 +51,13 @@ add-highlighter shared/git-commit/comments/ fill comment
 add-highlighter shared/git-commit/comments/ regex "\b(?:(modified)|(deleted)|(new file)|(renamed|copied)):([^\n]*)$" 1:yellow 2:red 3:green 4:blue 5:magenta
 }
 
+provide-module git-ignore %{
+add-highlighter shared/git-ignore group
++add-highlighter shared/git-ignore/glob regex '(?<!\\)(?:\\\\)*\K(\*\*?|\?|\[.*?(?<!\\)(?:\\\\)*\])' 0:operator
```

Comment:
That extra + at the beginning of the line looks weird

Reply:
my bad, I included that in my suggestion by accident

Reply:
This is a good catch. I didn't notice this. I just committed a fix.